hs_geographic_feature_resource/templates/pages/geographicfeatureresource.html

```diff
@@ -0,0 +1,235 @@
+{#                        {% endif %}#}
+{#                        </div>#}
+{#                    {% else %}#}
+{#                        {% if page.perms.change %}#}
+{#                            <div id="createNcdump" style="margin-bottom: 25px">#}
+{#                                <h3 style="margin-top:40px"> NetCDF Header Info</h3>#}
+{#                                <p>Click the button to create a new header info .txt file for the resource.</p>#}
+{#                                <a id="create-ncdump-btn" type="button" class="btn btn-success" href="{% url "nc_tools:create_ncdump" cm.short_id %}">#}
+{#                                    <span class="button-label"> Create header info file</span>#}
+{#                                </a>#}
+{#                            </div>#}
+{#                        {% endif %}#}
+{#                    {% endif %}#}
+{#                {% endfor %}#}
+{##}
+{#            </div>#}
+{#        {% endif %}#}
+
+
+    {% endblock %} {# extended block ends here #}
+
+
+{%  endblock %}
+
+
+{% block extra_js %}
+
+{{ block.super }}
+
+{# put resource specific js below #}
+<script type="text/javascript">
+    // original coverage meta check
+    function check_ori_meta_status(){
+        var form_action = $("form[id='id-originalcoverage']").attr('action');
+        if (form_action.indexOf('update-metadata') > -1) {
+            var new_action = form_action.replace('update-metadata', 'delete-metadata');
+            var modal_body_text = '<strong>Are you sure you want to delete this metadata element?</strong>'
+            $("#delete-original-coverage-element-dialog").find(".modal-body").html(modal_body_text);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").attr('href',new_action);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").show()
+        }
+        else {
+            var modal_body_text = '<strong>There is no original coverage metadata for this resource to delete.</strong>'
+            $("#delete-original-coverage-element-dialog").find(".modal-body").html(modal_body_text);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").hide()
+        }
+    };
+
+    del_msg_shp_shx_dbf = "<font color='red'>BE CAREFUL!</font> The 'shp', 'shx' and 'dbf' are three mandatory files. Missing of any of them will corrupt data integrity. So this operation is about to permanently <font color='red'>DELETE ALL EXISTING FILES</font> of this resource! You may upload new shapefiles later."
+    del_msg_prj = "This operation is about to remove coordinate system metadata!"
+    $(document).ready(function(){
+
+         $('.label-file-name').each( //iterate
+                 function()
+                 {
```

Comment:
the `function(){` doesn't need its own lines and can be pulled up into the line with the `each` call